include/bluetooth/gatt.h

```diff
@@ -181,6 +181,22 @@ struct bt_gatt_include {
 	uint16_t end_handle;
 };
 
+/** @brief GATT callback structure. */
+struct bt_gatt_cb {
+	/** @brief The maximum ATT MTU on a connection has changed.
+	 *
+	 *  This callback notifies the application that the maximum TX or RX
+	 *  ATT MTU has increased.
+	 *
+	 *  @param conn Connection object.
+	 *  @param tx Updated TX ATT MTU.
+	 *  @param rx Updated RX ATT MTU.
+	 */
+	void (*att_mtu_updated)(struct bt_conn *conn, uint16_t tx, uint16_t rx);
+
+	struct bt_gatt_cb *_next;
```

Comment:
Even though this is fairly trivial, I think we try to avoid custom linked list implementations and use `sys_snode_t` instead.